src/package/SlateEditor/SlateEditor.js

```diff
@@ -15,7 +15,11 @@ class SlateEditor extends Component {
   }
 
   onChange (state) {
-    this.setState({ state })
+    const { onChange } = this.props;
+    this.setState({ state });
+    if(onChange) {
```

Comment:
To call the `onChange` like a function below, may it would be better to check if the `onChange` prop is really a prop of function type... What do you think to do something like:

```js
onChange && onChange.constructor === Function && onChange(state)
```

And, if you can remove the semicolons it will be great too (: